src/sensor/ping.h

```diff
@@ -253,9 +253,9 @@ class Ping : public Sensor
      *
      * @param id
      */
-    void do_continuous_start(PingPing1DNamespace::ping_ping1D_id id)
+    void do_continuous_start(uint16_t id)
```

Comment:
This is a high level class, it's not save to use uint16_t as parameter, use enum.
The enum is explicit and allows the compiler to detect problems.

Reply:
done